src/SDL12_compat.c

```diff
@@ -8454,6 +8454,10 @@ SDL_LoadWAV_RW(SDL12_RWops *rwops12, int freerwops12,
     SDL_RWops *rwops20 = RWops12to20(rwops12);
     SDL_AudioSpec *retval = NULL;
 
+    if (!rwops20) {
```

Comment:
I wonder: Should you move this to after `*buf = NULL;` below?

Reply:
You're correct, that would be better. I had not considered that before. I'll change it.

What I also did not check there was the behavior of native SDL 1.2 with regards to buf; turns out that it is also left unitialized there. I don't know your policy there, so it's your call: Code that makes sense -> check after setting *buf, highest compatibility -> check before setting *buf. My take: I do not see a sensible way a caller might depend on the old SDL 1.2 behavior.